src/sys/stat.rs

```diff
@@ -123,8 +123,32 @@ pub fn fstatat<P: ?Sized + NixPath>(dirfd: RawFd, pathname: &P, f: AtFlags) -> R
     Ok(dst)
 }
 
+/// The chmod() function shall change S_ISUID, S_ISGID, S_ISVTX,
+/// and the file permission bits of the file named by the pathname
+/// pointed to by the path argument to the corresponding bits in the
+/// mode argument.
+///
+/// The fchmod() call cannot be used to portably change the
+/// permission bits of a file descriptor of a UNIX domain socket.
+///
+/// # References
+///
+/// [chmod(2)](http://pubs.opengroup.org/onlinepubs/9699919799/functions/chmod.html).
+pub fn chmod<P: ?Sized + NixPath>(path: &P, mode: Mode) -> Result<()> {
```

Comment:
Does directly binding `chmod(2)` have any advantages over `std::fs::set_permissions`?  It looks like that does the same thing.

Reply:
difference is that this uses the Mode flags, instead of the octal bits, which could be more descriptive. Still, it seems to offer similar functionality. Skip this method, or include it for completeness?

Reply:
Better not to bloat Nix if the method has no advantage.  Just ensure that Nix's `Mode` implements `Into<u32>` so it can be used with `std::fs::set_permissions`.

Reply:
Removed